src/api/spec/controllers/source_project_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
```

Comment:
Same here, if the tests broke with one of your previous commit, please fix the tests within that commit.

Reply:
I squashed everything in one commit. The changes are just meaningful as a whole